psp/src/alloc_impl.rs

```diff
@@ -44,7 +49,6 @@ unsafe impl GlobalAlloc for SystemAlloc {
 
         let id = *ptr.sub(align_padding as usize).cast::<SceUid>().offset(-1);
 
-        // TODO: Error handling.
```

Comment:
I have thought to add `log::error` enabled via some crate's feature in a similar place in case this function does ever fail. Just a suggestion.

Reply:
that's a really nice idea! I think that the std allocator just straight up ignore deallocation failures, but logging them would be a nice addition